cli/bench_command.go

```diff
@@ -80,6 +84,36 @@ type benchCmd struct {
 	filterSubject        string   // used by JS get command
 }
 
+// A sample for a particular client
+type sample struct {
```

Comment:
Please move these to an internal package. 

With all the many different commands in CLI package it’s hard to, for example, know what a “sample” is. 

But if they were in internal/bench/sample.go it would be much clearer what’s what and CLI tab completion in here would be cleaner for other commands